graphql/src/store/prefetch.rs

```diff
@@ -87,7 +86,7 @@ impl From<Option<q::TypeCondition>> for TypeCondition {
 /// has an entry mapping the response key to the list of nodes.
 #[derive(Debug, Clone)]
 struct Node {
-    entity: Entity,
+    entity: BTreeMap<String, q::Value>,
```

Comment:
We should really have a type name for this, since it's all over the codebase.

Reply:
We could call this `QueryValueObject` or something. But I can never recall what a type alias stands for when reading, so personally I'd keep it as it is.